cmd/cmd_utils.go

```diff
@@ -0,0 +1,87 @@
+package cmd
```

Comment:
shouldn't be this file called `cmdutils.go`?
Underscore in go source file names are usually used to separate tests or architectures `_linux.go`, `_win.go` ...
It looks a bit strange here, but maybe it is just me ;-)